packages/web-app-files/src/views/spaces/GenericSpace.vue

```diff
@@ -282,9 +282,16 @@ export default defineComponent({
     const isCurrentFolderEmpty = computed(
       () => unref(resourcesViewDefaults.paginatedResources).length < 1
     )
+    const isPublicFile = (spaceData: SpaceResource) => unref(spaceData).driveType === 'public'
```

Comment:
There is already function in web-client for this: https://github.com/owncloud/web/blob/9c4a40a20af9010d6e09937daae9784453b087d3/packages/web-client/src/helpers/space/types.ts#L98